Project.toml

```
@@ -18,7 +18,7 @@ ForwardDiff = "0.10.3"
 RecursiveArrayTools = "2"
 Reexport = "0.2"
 Setfield = "0.7"
-StaticArrays = "1.0"
+StaticArrays = "1.0 0.12"
```

```suggestion
StaticArrays = "0.12,1.0"
```

that's not well-formed?